knimin/handlers/ag_edit_participant.py

```diff
@@ -13,6 +13,9 @@ def get(self):
         email = self.get_argument('email', None)
         if email is not None:
             login = db.get_login_by_email(email)
+            if not login:
+                raise ValueError("Couldn't retrieve the login information for "
```

Comment:
Yes, this is the current standard - I do agree that it needs to change and provide more useful errors to the user.